edgelet/contrib/debian/postrm

```diff
@@ -3,22 +3,32 @@ set -e
 
 case "$1" in
     purge)
-        if [ -f /etc/aziot/edged/config.toml ]; then
-            rm /etc/aziot/edged/config.toml
-        fi
-        if [ -d /etc/aziot/edged/config.d ]; then
-            rm -r /etc/aziot/edged/config.d
-        fi
-        if [ -d /var/lib/aziot/edged ]; then
-            rm -rf /var/lib/aziot/edged
+        systemctl daemon-reload
+
+        if [ -d /etc/aziot ]; then
+            rm -rf /etc/aziot/edged
```

Comment:
Can you just delete `/etc/aziot/edged` and `/etc/aziot/config.toml` and let a-i-s's postrm handle the rest? Do postrm's run in a defined order so that a-i-s's will always run after this one?

Reply:
It's possible to uninstall aziot-edge without installing aziot-identity-service, right? Not that I would expect many people to do that, but currently this postrm is taking that into account.

Reply:
Although now that I think about it, yes, it's fine to just delete `/etc/aziot/edged/` and `/etc/aziot/config.toml`. Fixed.